teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super
+    @specialization = specialization
+  end
```

Comment:
- [ ] [REQUIRED] Good job so far 👍 I appreciate what you have done but in this `def` it will be clean if you do something like the method below too😃:

```ruby
...
  def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
...
```